network-api/networkapi/wagtailpages/models.py

```diff
@@ -788,7 +788,6 @@ class ParticipatePage2(PrimaryPage):
 
 class PeoplePage(PrimaryPage):
```

Comment:
If we no longer needs this `PeoplePage` we should fully remove this class.

Reply:
But only as a separate PR, because is a potentially problematic migration.

Reply:
I'll open a separate issue for this.